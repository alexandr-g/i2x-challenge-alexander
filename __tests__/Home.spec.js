/* env jest */

import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../src/components/home/Home';

describe('Home component', () => {
  test('is rendered', () => {
    const component = renderer.create(<Home />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
