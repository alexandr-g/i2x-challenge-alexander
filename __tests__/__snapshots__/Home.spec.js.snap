exports[`Home component is rendered 1`] = `
<div>
  <h2
    className="tc f6 fw6 ttu tracked">
    Get Started
  </h2>
  <p
    className="tc">
    Login to 
    <a
      className="f6 grow no-underline br-pill ba bw2 ph3 pv2 mb2 dib black"
      onClick={[Function]}
      style={Object {}}>
      demo app
    </a>
  </p>
</div>
`;
